### PR TITLE
WIP: assets: remove mongodb imagestreams and templates

### PR DESCRIPTION
**Pull request by yselkowitz**
These were deprecated in 4.6 but remained to support E2E testing. With
https://github.com/openshift/origin/pull/26072 merged, they are no longer
necessary.

### CODE REVIEW

**Comment by yselkowitz** [comment]
Once the origin PR merges, I'll use this is a pre-test before filing the openshift/library PR to remove these properly.

**Comment by gabemontero** [comment]
> Once the origin PR merges, I'll use this is a pre-test before filing the openshift/library PR to remove these properly.

sounds perfect - thanks @yselkowitz

**Comment by yselkowitz** [comment]
The latest CI payload shows the origin PR as being included, so let's give this a try now.
/test all

**Comment by gabemontero** [comment]
although it was marked failed, the image eco run https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-samples-operator/379/pull-ci-openshift-cluster-samples-operator-master-e2e-aws-image-ecosystem/1408128192283152384 had all the actual image eco tests pass @yselkowitz 

that is a good start 

let's see how 

/test e2e-aws-jenkins
/test e2e-aws-builds

do

**Comment by yselkowitz** [comment]
builds pass this time, let's try this one again:
/test e2e-aws-jenkins

**Comment by yselkowitz** [comment]
I'm a bit worried about this Jenkins test failure, did something slip through the origin PR testing (and how)?

**Comment by yselkowitz** [comment]
Posted a possible fix for the jenkins test, but since it wasn't caught in the big PR I'm not sure how to assure that actually works.

**Comment by gabemontero** [comment]
I have some meetings this AM I need to focus on first but afterwards I'll spend time on the e2e-aws-jenkins failure here, as well as your new openshift/origin PR, and catch up / connect dots where I can.

**Comment by gabemontero** [comment]
OK, first, looking at https://github.com/openshift/origin/pull/26072, I see why we missed this.  In all the flake churn there, I missed that we never ran e2e-gcp-jenkins / e2e-aws-jenkins

the e2e-gcp-builds suite only runs a subset of the jenkins suite.  the full jenkins suite has more expensive / slower tests that we had to isolate to keep the builds e2e running time and resource needs with certain confine.

If you click the "View details" button from where Michal merged it, you'll see there is no "e2e-*-jenkins" run.

That explains the why we missed it.   Sigh...

I'll jump over to your new openshift/origin PR.

**Comment by yselkowitz** [comment]
Let's see how 4.9 tests look now
/test all

**Comment by yselkowitz** [comment]
Should I be concerned with the OKD failures or are they just generally broken again?

**Comment by gabemontero** [comment]
the okd-e2e-aws-image-ecosystem is interesting because of this failure there:

```


:  [sig-devex][Feature:ImageEcosystem][Slow] openshift sample application  repositories [sig-devex][Feature:ImageEcosystem][php] test php images  with cakephp-ex db repo  Building cakephp-mysql app from new-app should  build a cakephp-mysql image and run it in a pod expand_less | 2m43s
-- | --
fail [github.com/openshift/origin/test/extended/image_ecosystem/sample_repos.go:57]: Unexpected error:     <*errors.errorString \| 0xc00269c120>: {         s: "Failed to import expected imagestreams, latest error status: ImageStream Error: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:\"\", APIVersion:\"\"}, ListMeta:v1.ListMeta{SelfLink:\"\", ResourceVersion:\"\", Continue:\"\", RemainingItemCount:(*int64)(nil)}, Status:\"Failure\", Message:\"imagestreams.image.openshift.io \\\"mongodb\\\" not found\", Reason:\"NotFound\", Details:(*v1.StatusDetails)(0xc002d7d4a0), Code:404}} \n",     }     Failed to import expected imagestreams, latest error status: ImageStream Error: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:"", Continue:"", RemainingItemCount:(*int64)(nil)}, Status:"Failure", Message:"imagestreams.image.openshift.io \"mongodb\" not found", Reason:"NotFound", Details:(*v1.StatusDetails)(0xc002d7d4a0), Code:404}}       occurred

```

but I've seen this before, in that the origin level pulled in for the okd tests is not kept as frequently up to date as the OCP version, hence the reference still to mongodb, even though that is no longer present in master branch per https://github.com/openshift/origin/blob/c64033b5c432afa340f95cd9abd9219874bb11ba/test/extended/util/framework.go#L242

seeing similar out of date refs to mongodb in the okd jenkins e2e

bottom line, I think we can consider your litmus test successful @yselkowitz because ocp build/image-eco/jenkins are green

go ahead and close this out and proceed with you openshift/library branch

thanks 

@dperaza4dustbit FYI ... more fodder for our next education session ... this PR exhibits the legitimate means of validating huge changes in the SCL samples and the corresponding OCP e2e's out of openshift/origin
